Bender.yml

```diff
@@ -29,7 +29,8 @@ dependencies:
   tech_cells_generic: { git: https://github.com/pulp-platform/tech_cells_generic, version:  0.2.11  }
   riscv-dbg:          { git: https://github.com/pulp-platform/riscv-dbg,          version:  0.8.0   }
   hwpe-mac-engine:    { git: https://github.com/KULeuven-MICAS/hwpe-mac-engine.git, rev: 5d3b4525b665169fc8321c8a811f3c83ad3c72e8 }
-  snax-gemm: { git: "https://github.com/KULeuven-MICAS/snax-gemm", rev: 5a57b66e9b29f51de77dac99fd466c6aee073175 }
+  snax-gemm: { git: "https://github.com/KULeuven-MICAS/snax-gemm", rev: 5a57b66e9b29f51de77dac99fd466c6aee073175  }
+  snax-postprocessing-simd: { git: "https://github.com/KULeuven-MICAS/snax-postprocessing-simd", rev: 889b2d9f348368a912dbc3183c65aaa27a7f0c9b }
```

Comment:
We are not pointing to the main or something? It's really a specific revision? 😄

Reply:
Changed!